### PR TITLE
refine circuit breaker ut to avoid massive log

### PR DESCRIPTION
**Pull request by louyuting**
### Describe what this PR does / why we need it
refine circuit breaker ut to avoid massive log

### Describe how to verify it
UT

### Special notes for reviews

### CODE REVIEW

File: tests/core/circuitbreaker/circuitbreaker_slot_integration_test.go [hunk 1]
```diff
@@ -25,18 +25,18 @@ type StateChangeListenerMock struct {
...
 	_ = s.Called(prev, rule)
-	fmt.Printf("rule.steategy: %+v, From %s to Closed, time: %d\n", rule.Strategy, prev.String(), util.CurrentTimeMillis())
+	logging.Debugf("rule.steategy: %+v, From %s to Closed, time: %d\n", rule.Strategy, prev.String(), util.CurrentTimeMillis())
 	return
 }
 
 func (s *StateChangeListenerMock) OnTransformToOpen(prev circuitbreaker.State, rule circuitbreaker.Rule, snapshot interface{}) {
 	_ = s.Called(prev, rule, snapshot)
-	fmt.Printf("rule.steategy: %+v, From %s to Open, snapshot: %.2f, time: %d\n", rule.Strategy, prev.String(), snapshot, util.CurrentTimeMillis())
+	logging.Debugf("rule.steategy: %+v, From %s to Open, snapshot: %.2f, time: %d\n", rule.Strategy, prev.String(), snapshot, util.CurrentTimeMillis())
```
Comment (sczyh30) [inline tests/core/circuitbreaker/circuitbreaker_slot_integration_test.go:34]: steategy -> strategy?

Comment (louyuting) [reply tests/core/circuitbreaker/circuitbreaker_slot_integration_test.go:34]: updated

**Review by sczyh30 — APPROVED** [review]
LGTM

**Comment by sczyh30** [comment]
Thanks!

**Pull request merged**